pkg/virt-operator/resource/apply/core_test.go

```diff
@@ -266,7 +274,89 @@ var _ = Describe("Apply", func() {
 
 			updatedCrt := createCrt()
 
-			_, err = r.createOrUpdateKubeVirtCAConfigMap(queue, updatedCrt, duration, requiredCM)
+			_, err = r.createOrUpdateKubeVirtCAConfigMap(queue, updatedCrt, nil, duration, requiredCM)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(patched).To(BeTrue())
+		})
+
+		It("should create ConfigMap when it doesn't exist", func() {
+			requiredCMs := components.NewCAConfigMaps(operatorNamespace)
+			var requiredCM *corev1.ConfigMap
+			for _, cm := range requiredCMs {
+				if cm.Name == components.KubeVirtCASecretName {
+					requiredCM = cm
+				}
+			}
+			r := &Reconciler{
+				kv:           kv,
+				stores:       stores,
+				clientset:    clientset,
+				expectations: expectations,
+			}
+			created := false
+			coreclientset.Fake.PrependReactor("create", "configmaps", func(action testing.Action) (handled bool, ret runtime.Object, err error) {
+				created = true
+				return true, &corev1.ConfigMap{}, nil
+			})
```

Comment:
Instead of prepending a reactor, can't you check the object exists in the fake client after it was created?

Reply:
This is to stop the check below from saying hey I have received a call to create that I was not expecting.